core/src/components/progress-bar/test/basic/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html dir="ltr">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Progress Bar - Basic</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <link href="../../../../../css/ionic.bundle.css" rel="stylesheet">
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet">
+    <script src="../../../../../dist/ionic.js"></script>
+    <style>
+        .custom-bar-background {
+            background: red;
+        }
+
+        .no-bar-background {
+            background: none;
+        }
+
+        ion-progress-bar {
+            margin: 10px 0;
```

Comment:
I think it's important for ionic component to look nice by default, should we include this CSS in the host element, directly? it could still be override.

Reply:
The question is where to set the progress bar - if I want it at the top, then it would be great to have no margin by default - and in my use cases I dont have a margin for a progress bar